keras_cv/models/densenet.py

```diff
@@ -0,0 +1,296 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DenseNet models for KerasCV.
+
+Reference:
+  - [Densely Connected Convolutional Networks](https://arxiv.org/abs/1608.06993)
+"""
```

Comment:
If it is referred from the `keras.applications` code, do we need to cite that to avoid self-plagiarism issues?

Reply:
+1, let's do that